### ISSUE TITLE
Price slider incorrect format in german locale store

### ISSUE THREAD

**Issue by tim-breitenstein-it** [magesuite/theme-creativeshop#59]
The javascrip parseFloat() function is not working for price slider if the store locale is german.

In a german store the `,` is the decimal separator und the `.` is the thousand separator.

URL: `http://m24.test.com/test.html?price=0-4870`

Initial situation:

![image](https://user-images.githubusercontent.com/65597245/108509026-fd0f7980-72bc-11eb-8506-08abfff90be2.png)

Click into from price (1) => to price change to incorret format

![image](https://user-images.githubusercontent.com/65597245/108509041-00a30080-72bd-11eb-945c-2c811112ede1.png)

Click into to price (2) => to price change to incorret format

![image](https://user-images.githubusercontent.com/65597245/108509053-039df100-72bd-11eb-9f17-a64ff17e5576.png)

Same behavior for to price if to price is filled.

In english locale store it works fine benause the `.` is the decimal separator und the `,` the thousand separator.

![image](https://user-images.githubusercontent.com/65597245/108515024-e5d48a00-72c4-11eb-9c3b-52e5a0f869f2.png)

Solution for German store:

```
_onInputsChange: function(event) {
                var key = event.key || event.keyCode;

                if (event.type === 'blur' || key === 'Enter' || key === 13) {
                    var from = parseFloat(
                        this.element
                            .find(this.options.fromInput)
                            .val()
                            .replace(/[^\d\.\,]/, '')
                            .replace(/\./g, '')
                            .replace(/\,/g, '.') || 0
                    );
                    var to = parseFloat(
                        this.element
                            .find(this.options.toInput)
                            .val()
                            .replace(/[^\d\.\,]/, '')
                            .replace(/\./g, '')
                            .replace(/\,/g, '.') || 0
                    );

                    if (!to) {
                        to = this.maxValue;
                    }

                    if (from > to) {
                        from = to - this.rate;
                    }

                    if (from < this.minValue) {
                        from = this.minValue;
                    }

                    if (to > this.maxValue) {
                        to = this.maxValue;
                    }

                    this._updateRange(from, to);
                }
            }
```
If switch back to english store the solution for german store will not work.

**Comment by mborkowski** [comment]
Hi @tim-breitenstein-it 
Nice catch. Sorry for late reply but I had to wait for QA result. So I can confirm the issue, however reproduction path seems to be a bit different so I'm not sure if we're talking about the same thing. Could you please check if the same thing happens gor you after you simply focus/blur an input? 
Second: we'd be grateful for a PR covering all scenarios (not only german case). If, for any reason you can't or simply don't want to contribute, please let me know so I could create internal ticket.

**Comment by tim-breitenstein-it** [comment]
I implemented a solution based on:

- [globalize](https://github.com/globalizejs/globalize)
- [cldrjs](https://github.com/rxaviers/cldrjs)

Get the actual language based on `<html lang="de">`.

But I think it`s a bit overloaded for this small feature? What you think?

```
    let lang = document.querySelector('html').getAttribute('lang') || 'en';

    Globalize.locale(lang);
    let numberParser = Globalize.numberParser();

...
            _onInputsChange: function(event) {
...
                    var from = numberParser(this.element.find(this.options.fromInput).val());
                    var to = numberParser(this.element.find(this.options.toInput).val());
```

**Comment by mborkowski** [comment]
Yeah, I'd rather play around with [toLocaleString](https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) but then we have to parseInt those values etc, so it has to work 2-way (read and write). We created internal ticket for this but I wouldn't expect this fix soonish 😞 
If your fix works and you're ok with it - please leave it as customization on your side while waiting for the proper fix

**Comment by ptylek** [comment]
@tim-breitenstein-it we prepared [solution](https://github.com/magesuite/theme-creativeshop/commit/2ead1705c36e8d9b6be6a509ee9d08efa081cb53) and it should be available with next release

**Comment by tim-breitenstein-it** [comment]
Seems to work!
